client/src/main/java/org/apache/rocketmq/client/consumer/DefaultLitePullConsumer.java

```diff
@@ -271,6 +271,11 @@ public void assign(Collection<MessageQueue> messageQueues) {
         defaultLitePullConsumerImpl.assign(queuesWithNamespace(messageQueues));
     }
 
+    @Override
+    public void setSubExpression4Assign(final String topic, final String subExpresion) {
```

Comment:
Naming is inconsistent with the existing style

Reply:
do u have any suggestion for this naming?

Reply:
Use full words to make it easy to follow. No non-standard, obscure, abbreviations.